geomstats/euclidean_space.py

```diff
@@ -29,20 +30,18 @@ def belongs(self, point):
 
     def random_uniform(self, n_samples=1):
         """
-        Sample a vector uniformly in the Euclidean space,
-        with coordinates each between -1. and 1.
+        Sample in the Euclidean space with the uniform distribution.
         """
         size = (n_samples, self.dimension)
-        point = gs.random.rand(*size) * 2 - 1
+        point = (gs.random.rand(*size) - 0.5) * 2
 
         return point
 
 
 class EuclideanMetric(RiemannianMetric):
     """
-    Class for the Euclidean metric.
-    The metric is flat: inner product independent of the reference point.
-    The metric has signature (0, n) on the n-D vector space.
+    Class for Euclidean metrics.
```

Comment:
that's litterally what's written in the code. Either remove or put something more precise.